### PR TITLE
Add Anita as maintainer, changes to DOI, PMID

### PR DESCRIPTION
**Pull request by matentzn**
1. Revert DOI to default
2. fix PMID prefix in the OBO Foundry standard manner

cc @anitacaron

### CODE REVIEW

**Review by anitacaron — APPROVED** [review]

File: src/bioregistry/data/contexts.json [hunk 1]
```diff
@@ -13,7 +13,12 @@
         "github": "matentzn",
         "name": "Nico Matentzoglu",
         "orcid": "0000-0002-7356-1779"
-      }
+      },
+      {
+        "github": "anitacaron",
+        "name": "Anita Caron",
+        "orcid": "0000-0002-6523-4866"
+      },
```
Comment (matentzn) [inline src/bioregistry/data/contexts.json:21]:
```suggestion
      }
```

Suggestion from @gouttegd

File: src/bioregistry/data/contexts.json [hunk 2]
```diff
@@ -38,4 +43,4 @@
       "ols"
     ]
   }
-}
\ No newline at end of file
+}
```
Comment (gouttegd) [inline src/bioregistry/data/contexts.json:46]: The extra new line character after the final closing bracket needs removing. That’s what is causing the linter test to fail.

Comment (gouttegd) [reply src/bioregistry/data/contexts.json:46]: Actually that’s not enough, the keys must also be sorted for the linter to be happy.

Comment (gouttegd) [reply src/bioregistry/data/contexts.json:46]: Updated PR in #906.

Comment (matentzn) [reply src/bioregistry/data/contexts.json:46]: Thanks!

**Comment by matentzn** [comment]
Replaced by #906

**Pull request closed without merging**